src/resources/Enums.ts

```diff
@@ -132,43 +132,34 @@ export enum SourceStatusTypeWithTransition {
 }
 
 export enum SourceType {
-    ADOBE_EXPERIENCE_MANAGER = 'ADOBE_EXPERIENCE_MANAGER',
+    ADOBE_EXPERIENCE_MANAGER = 'ADOBE_EXPERIENCE_MANAGER', // @deprecated
     AMAZONS3 = 'AMAZONS3',
-    BOX = 'BOX',
-    BOX_ENTERPRISE = 'BOX_ENTERPRISE',
     BOX_ENTERPRISE2 = 'BOX_ENTERPRISE2',
     CATALOG = 'CATALOG',
-    CONFLUENCE = 'CONFLUENCE',
+    CONFLUENCE = 'CONFLUENCE', // @deprecated
     CONFLUENCE2 = 'CONFLUENCE2',
     CONFLUENCE2_HOSTED = 'CONFLUENCE2_HOSTED',
     CUSTOM = 'CUSTOM',
     DATABASE = 'DATABASE',
-    DROPBOX = 'DROPBOX',
     DROPBOX_FOR_BUSINESS = 'DROPBOX_FOR_BUSINESS',
-    EXCHANGE = 'EXCHANGE',
-    EXCHANGE_ENTERPRISE = 'EXCHANGE_ENTERPRISE',
     FILE = 'FILE',
-    GENERIC = 'GENERIC_REST',
-    GMAIL = 'GMAIL',
-    GMAIL_DOMAIN_WIDE = 'GMAIL_DOMAIN_WIDE',
-    GMAIL_SINGLE_USER = 'GMAIL_SINGLE_USER',
+    GENERIC_REST = 'GENERIC_REST',
     GOOGLE_DRIVE_DOMAIN_WIDE = 'GOOGLE_DRIVE_DOMAIN_WIDE',
-    GOOGLE_DRIVE_SINGLE_USER = 'GOOGLE_DRIVE_SINGLE_USER',
     GRAPHQL = 'GRAPHQL',
     JIRA2 = 'JIRA2',
     JIRA2_HOSTED = 'JIRA2_HOSTED',
     JIVE = 'JIVE',
     JIVE_HOSTED = 'JIVE_HOSTED',
     KHOROS = 'KHOROS',
     LITHIUM = 'LITHIUM',
-    MICROSOFT = 'MICROSOFT_DYNAMICS',
-    ONEDRIVE = 'ONEDRIVE_FOR_BUSINESS',
+    MICROSOFT_DYNAMICS = 'MICROSOFT_DYNAMICS',
```

Comment:
This one changed name, it could be breaking

Reply:
Yes, but the value remains unchanged and I'm standardizing the keys as for the rest of the enum. The correct data and the one that the API accepts remain unchanged, it's still the value

Reply:
As we've just been talking about it, I'm simply going to write a nice commit message to make the changes made in this PR very explicit.